yangvalidator/v1/views.py

```diff
@@ -186,35 +184,35 @@ def copy_dependencies(f):
     yang_models = config.get('Directory-Section', 'save-file-dir')
     temp_dir = config.get('Directory-Section', 'temp')
     out = f.getvalue()
-    logger.info('dependencies received in following format: {}'.format(out))
+    logger.info(f'dependencies received in following format: {out}')
     letters = string.ascii_letters
     suffix = ''.join(random.choice(letters) for i in range(8))
-    dep_dir = '{}/yangvalidator-dependencies-{}'.format(temp_dir, suffix)
+    dep_dir = f'{temp_dir}/yangvalidator-dependencies-{suffix}'
     os.mkdir(dep_dir)
     if len(out.split(':')) == 2:
         dependencies = out.split(':')[1].strip().split(' ')
     else:
         dependencies = []
     for dep in dependencies:
-        for file in glob.glob(r'{}/{}@*.yang'.format(yang_models, dep)):
+        for file in glob.glob(rf'{yang_models}/{dep}@*.yang'):
             shutil.copy(file, dep_dir)
     return dep_dir
 
 
 def validate_yangfile(infilename, workdir):
-    logger.info('validating {}'.format(infilename))
+    logger.info(f'validating {infilename}')
     pyang_res = {}
     yanglint_res = {}
     confdc_res = {}
     yangdump_res = {}
     confdc_output = yanglint_output = confdc_stderr = yanglint_stderr = yangdump_output = yangdump_stderr = ''
     infile = os.path.join(workdir, infilename)
-    confdc_resfile = str(os.path.join(workdir, infilename) + '.cres')
-    confdc_outfile = str(os.path.join(workdir, infilename) + '.cout')
-    yanglint_resfile = str(os.path.join(workdir, infilename) + '.lres')
-    yanglint_outfile = str(os.path.join(workdir, infilename) + '.lout')
-    yangdump_resfile = str(os.path.join(workdir, infilename) + '.ypres')
-    yangdump_outfile = str(os.path.join(workdir, infilename) + '.ypout')
+    confdc_resfile = f'{os.path.join(workdir, infilename)}.cres'
```

Comment:
Result of `os.path.join(workdir, infilename)` is stored in `infile` variable.
Can we use it instead of `{os.path.join(workdir, infilename)`?
Same for all variable bellow